### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Sem-the-dev**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -3,9 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=H, initial-scale=1.0">
-    <title>Document</title>
+    <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="static/css/normalize.css">
+    <link rel="stylesheet" href="static/css/skeleton.css">
```
Comment (AdilIqbal95) [inline client/index.html:8]: Nice use of a new library

File: client/index.html [hunk 2]
```diff
@@ -3,9 +3,28 @@
...
+    <section>
+    <form>
+        <div class="row">
+        <input type="text" name="title" placeholder="Enter a title"></div>
+        <div class="row">
+        <input type="text" name="username" placeholder="Enter a psuedoname"></div>
+        <div class="row">
+        <textarea name="body" placeholder="Type message" width="100" height="50"></textarea></div>
+        <div class="row">
+        <input type="submit" value="publish">
```
Comment (AdilIqbal95) [inline client/index.html:24]: Nicely laid out page. Good creation of the form.

File: client/static/css/normalize.css [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+td,
+th {
+  padding: 0;
+}
```
Comment (AdilIqbal95) [inline client/static/css/normalize.css:429]: - Clearly laid out CSS
- Nice use of commenting made it easy to read
- Great job making it responsive

File: client/static/css/skeleton.css [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+@media (min-width: 550px) {}
+
+/* Larger than tablet */
+@media (min-width: 750px) {}
+
+/* Larger than desktop */
+@media (min-width: 1000px) {}
+
+/* Larger than Desktop HD */
+@media (min-width: 1200px) {}
```
Comment (AdilIqbal95) [inline client/static/css/skeleton.css:419]: - Good use of media queries!
- Nice use of the skeleton framework

File: client/static/js/index.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        const postSection = document.createElement('div');
+        const title = document.createElement('h3');
+        const body = document.createElement('p');
+        title.textContent = postData.title;
+        body.textContent = postData.body;
+        postSection.append(title);
+        postSection.append(body);
+    }
+    
+    newPost.append(postSection);
```
Comment (AdilIqbal95) [inline client/static/js/index.js:60]: - Nice use of async functions
- Good job with the posts routing & storing

File: server/models/Post.js [hunk 6]
```diff
@@ -56,6 +61,7 @@ class Post {
                                                     FROM posts
                                                     WHERE posts.unique_id = ${special_id}`);
                 let post = new Post(result.rows[0])
+                console.log(post)
                 res(post)
```
Comment (AdilIqbal95) [inline server/models/Post.js:65]: - Nice flow in your functions (indentation, commenting etc)
- This file (models/posts) is different to what we seemed to have cloned down - not sure if we did something wrong here.

**Comment by Sem-the-dev** [comment]
Thank you for the feedback :)

**Review by AdilIqbal95 — APPROVED** [review]
- Good overall product - you've completed all the necessary requirements.
- It was good to see that you were thinking of mobile users.
- We couldn't quite tell if the post is stored in the database?

For next time:
- Would be good to try and implement the 'id' route.
- Although it was interesting to see the different styling - maybe it was too bulky?

**Pull request merged**